rasa/shared/core/training_data/visualization.py

```diff
@@ -402,20 +400,40 @@ def _add_message_edge(
     is_current: bool,
 ) -> None:
     """Create an edge based on the user message."""
-
     if message:
-        message_key = message.get("intent", {}).get("name", None)
+        message_intent = message.get("intent", {}).get("name", None)
+        # get the entities as \n separated values
+        entities: Optional[List[Dict[Text, Any]]] = message.get("entities")
+        entity_values = []
+        if entities:
+            entity_values = [
+                value
+                for entity in entities
+                for key, value in entity.items()
+                if key == "entity"
+            ]
+        if entity_values:
+            newline = ",\n"
+            entity_values_as_str = f"{newline}({newline.join(entity_values)})"
+        else:
+            entity_values_as_str = ""
+
+        message_key = message_intent + entity_values_as_str
+        message_label = message_key  # message.get("text", None)
```

Comment:
What does this comment represent? Is it supposed to be removed?

Reply:
I just realised, what was the need here, i believe this is to make sure that the label of the edge, shows the intent and entity values as str, but according to me, it is fine using the text as label as well, reverted this.

Reply:
i suppose i understand, why seeing the entities would make sense here. removed the comment

Reply:
Thanks, I was just about to comment on the comment. It is not the entity values, it is the entity keys, maybe renaming the variable would help here.